infra/examples/aws-msft-365/main.tf

```diff
@@ -194,3 +194,15 @@ module "msft_365_grants" {
   app_roles                = each.value.required_app_roles
   application_name         = each.key
 }
+
+
+module "worklytics-psoxy-connection" {
+  for_each = local.msft_sources
```

Comment:
I'd add a `deploy: bool` to each and then filter the ones with only deploy.
Is easier for the customer to just flip a switch than commenting full connector settings.

https://github.com/Worklytics/psoxy/blob/522147ac2b69553d8f2f08d45f44f7dc21d7e0eb/infra/examples/aws-zoom-slack/main.tf#L59